src/js/index.js

```diff
@@ -1,2 +1,69 @@
 import 'bootstrap/dist/css/bootstrap';
 import $ from 'jquery';
+
+import requestAdaptor from './request.js';
+
+const request = new requestAdaptor();
+
+const main = function() {
```

Comment:
`main` is not a very good function name. Hard to guess what this function is made for

Reply:
I guess this is from Java lol